docs/_includes/default.html

```diff
@@ -1,51 +1,51 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
So, something is overwriting this change, and I think it's an old `html-minifier` version. This was fixed a few versions ago, but I don't have the time to track which dep (I think it's that assetgraph-builder one).

Reply:
Because it's [wrong](https://www.w3.org/TR/html-polyglot/#doctype)...

Reply:
No it's not wrong. You are confusing things. That's for XML.

https://mathiasbynens.be/notes/xhtml5#doctype

Reply:
Even your own reference says it's incorrect.

Reply:
I don't think you even read what I sent. That XHTML, not HTML, which is what you are using.

Reply:
Sure I did
> If you insist on using it, however, you should know ‘DOCTYPE’ is supposed to be written in uppercase

 but missed part
> in XML mode.

Reply:
Nonetheless, not accepting _any_ change going against W3C...

Reply:
Seriously, you don't see that this is HTML, not XHTML?

You keep saying something that's wrong.

Reply:
>  Polyglot markup uses a document type declaration (DOCTYPE) specified by section 8.1.1 of [HTML5]. In addition, the DOCTYPE conforms to the following rules:
>
>    The string DOCTYPE is in uppercase letters.

Reply:
@plroebuck The spec you're referring to is out-of-date, but importantly, "polyglot" markup is defined as markup that is both valid HTML5 and XHTML (I think this is referred to as "XHTML5") at once.  Our site is not claiming to be XHTML-compatible.

Nowadays, browser vendors work with/from the WHATWG's HTML living standard.  Here's what it says about [doctype](https://html.spec.whatwg.org/#syntax-doctype).